app/views/application/_head.html.erb

```diff
@@ -20,12 +20,34 @@
     rel: "mask-icon"
   ) %>
   <link rel="stylesheet" href="https://use.typekit.net/zmv8alo.css">
-  <script>L_DISABLE_3D = true;</script>
+
   <%= stylesheet_link_tag 'application', 'data-turbolinks-track': 'reload', media: 'all' %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
 
-  <% if ENV["FEEDBACK_FISH_ID"].present? %>
-    <script defer src="https://feedback.fish/ff.js?pid=<%= ENV['FEEDBACK_FISH_ID'] %>"></script>
+  <% if feedback_fish_id = ENV["FEEDBACK_FISH_ID"] %>
+    <%= tag.script(src: "https://feedback.fish/ff.js?pid=#{feedback_fish_id}", defer: "defer", nonce: request.content_security_policy_nonce) %>
+  <% end %>
+
+  <% if gtmid = ENV["GOOGLE_TAG_MANAGER_ID"] %>
```

Comment:
Could we put these environment variables into configuration settings instead of reading `ENV` directly from a view?